x-pack/plugins/watcher/public/application/sections/watch_list_page/watch_list_page.tsx

```diff
@@ -275,42 +276,99 @@ export const WatchListPage = () => {
+        ),
         sortable: true,
         width: '130px',
-        render: (state: string) => <WatchStatus status={state} />,
+        render: (state: string) => <WatchStateBadge state={state} />,
       },
       {
         field: 'watchStatus.lastMetCondition',
-        name: i18n.translate('xpack.watcher.sections.watchList.watchTable.lastFiredHeader', {
-          defaultMessage: 'Last fired',
-        }),
+        name: (
+          <EuiToolTip
+            content={i18n.translate(
+              'xpack.watcher.sections.watchList.watchTable.lastFiredHeader.tooltipText',
+              {
+                defaultMessage: `The last time the condition was met and action taken.`,
+              }
+            )}
+          >
+            <span>
+              {i18n.translate('xpack.watcher.sections.watchList.watchTable.lastFiredHeader', {
+                defaultMessage: 'Condition last met',
+              })}{' '}
+              <EuiIcon size="s" color="subdued" type="questionInCircle" className="eui-alignTop" />
+            </span>
+          </EuiToolTip>
+        ),
         sortable: true,
         truncateText: true,
-        width: '130px',
+        width: '160px',
         render: (lastMetCondition: Moment) => {
           return lastMetCondition ? lastMetCondition.fromNow() : lastMetCondition;
         },
       },
       {
         field: 'watchStatus.lastChecked',
-        name: i18n.translate('xpack.watcher.sections.watchList.watchTable.lastTriggeredHeader', {
-          defaultMessage: 'Last triggered',
-        }),
+        name: (
+          <EuiToolTip
+            content={i18n.translate(
+              'xpack.watcher.sections.watchList.watchTable.lastTriggeredHeader.tooltipText',
+              {
+                defaultMessage: `The last time the condition was checked.`,
+              }
+            )}
+          >
+            <span>
+              {i18n.translate('xpack.watcher.sections.watchList.watchTable.lastTriggeredHeader', {
+                defaultMessage: 'Last checked',
+              })}{' '}
+              <EuiIcon size="s" color="subdued" type="questionInCircle" className="eui-alignTop" />
+            </span>
+          </EuiToolTip>
+        ),
         sortable: true,
         truncateText: true,
-        width: '130px',
+        width: '160px',
         render: (lastChecked: Moment) => {
           return lastChecked ? lastChecked.fromNow() : lastChecked;
         },
       },
       {
         field: 'watchStatus.comment',
-        name: i18n.translate('xpack.watcher.sections.watchList.watchTable.commentHeader', {
-          defaultMessage: 'Comment',
-        }),
+        name: (
+          <EuiToolTip
+            content={i18n.translate(
+              'xpack.watcher.sections.watchList.watchTable.commentHeader.tooltipText',
+              {
+                defaultMessage:
+                  'Whether any actions are acknowledgeable or have been acknowledged, throttled, or failed to execute.',
```

Comment:
Sorry, I missed this one before.  I'm not sure what "are ackledgeable" means.  Can this be:

> Whether actions are acknowledged, throttled, or failed.

or

> Whether actions are in an acknowledged, throttled, or failed state.